setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='biocrnpyler',
-    version='1.1.0',
+    version='1.1.1',
```

Comment:
why does this PR update the version? we can do this in dev-master PR

Reply:
good point - fixed on next commit.